Source/Configuration/YPIcons.swift

```diff
@@ -26,4 +26,5 @@ public struct YPIcons {
     public var captureVideoImage: UIImage = imageFromBundle("yp_iconVideoCapture")
     public var captureVideoOnImage: UIImage = imageFromBundle("yp_iconVideoCaptureRecording")
     public var playImage: UIImage = imageFromBundle("yp_play")
+    public var removeImage: UIImage = imageFromBundle("yp_remove")
```

Comment:
Please change the filename of yp_remove from `Remove-universal-32@..png` to `yp_remove@..png`